docs/installation/installation.md

```diff
@@ -13,4 +13,4 @@ We also have a [VSCode extension](/installation/vscode-extension)
 
 ![Move2Kube Installation]({{ site.baseurl }}/assets/images/install.png)
 
-You can try out Move2Kube in [Katacoda](https://www.katacoda.com/move2kube).
+You can try out Move2Kube in [Katacoda](https://move2kube.konveyor.io).
```

Comment:
Similarly, we should also remove the text and links related to Katacoda, instead of redirecting `Move2Kube on Katacoda` to https://move2kube.konveyor.io.